### PR TITLE
Modal Window

### PR DESCRIPTION
**Pull request by sae-github**
# Modal Window

### Specification Description
-Click on the button to open the modal window.
Also Click on the black background and the Close button to close the modal window.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  text-align: center;
+  opacity: 0;
+  overflow: hidden;
+  z-index: -1;
+  transition: opacity .3s;
+}
+
+
+.p-modal.is-show,
+.p-overlay.is-show {
```
Comment (kenmori) [inline src/style.css]: Is this no good?

```suggestion
.is-show, .is-show {
```

**Comment by kenmori** [comment]
I want it to be in the vertical center. you think so too?

スクリーンショット 2021-07-27 18 56 04

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  overLay.classList.add('is-show');
+});
+closeBtn.addEventListener('click',function() {
+  modal.classList.remove('is-show');
+  overLay.classList.remove('is-show');
+});
+overLay.addEventListener('click',function() {
+  this.classList.remove('is-show');
+  modal.classList.remove('is-show');
+});
```
Comment (kenmori) [inline src/index.js]: The close button and the button in the overlay behave the same, but wasn't one?

**Comment by kenmori** [comment]
IMO
I think this implementation is good, but I tried to make it a little simpler. Please [refer to it](https://codesandbox.io/s/modal-windowjs-forked-x5po7?file=/src/index.js) if you like.
I know. my CSS is dirty and should be a little better.

File: src/style.css (hunk 1 above)
Comment (sae-github) [reply src/style.css]: Thank you.
I thought it was better that way.

**Comment by sae-github** [comment]
> I want it to be in the vertical center. you think so too?
> 
> スクリーンショット 2021-07-27 18 56 04

Thank you.
Fixed.
I looked at a few reference sites.For example, [this](https://material.io/components/dialogs).
In many cases, they were installed in the center of the top and bottom.

**Comment by sae-github** [comment]
> think this implementation is good, but I tried to make it a little simpler. Please [refer to it](https://codesandbox.io/s/modal-windowjs-forked-x5po7?file=/src/index.js) if you like.
> I know. my CSS is dirty and should be a little better.

Thank you.
I have made the correction based on [this.](https://codesandbox.io/s/modal-windowjs-41lkl?file=/index.html)
I thought using transition would make the movement smoother. But I am not sure if this is the best way.
What do you think? 57c83af

File: src/index.js (hunk 2 above)
Comment (sae-github) [reply src/index.js]: Same movement.
I should have enclosed it in the same parent element.

**Comment by kenmori** [comment]
nits
スクリーンショット 2021-07-28 22 36 40

**Comment by kenmori** [comment]
> What do you think?

That's good too.

**Comment by sae-github** [comment]
> > What do you think?
> 
> That's good too.

Thanks for the review.

**Comment by sae-github** [comment]
> nits
> スクリーンショット 2021-07-28 22 36 40


Fixed.

**Pull request merged**